yb-voyager/cmd/live_migration.go

```diff
@@ -55,6 +56,10 @@ func streamChangesFromSegment(segment *EventQueueSegment) error {
 	}
 	defer segment.Close()
 	log.Infof("streaming changes for segment %s", segment.FilePath)
+	valueConverter, err = dbzm.NewValueConverter(exportDir, tdb, false) //streaming valueConverter
```

Comment:
Once you drop the `snapshotMode`, you won't need to re-instantiate a new converter.

Reply:
Oh yeah!